EBH-4608/concat_excel.py

```
@@ -0,0 +1,190 @@
+import openpyxl
+import argparse
+import subprocess
+import re
+import io as io
+
+project_id = "project-J4V0y18454QjYbJjzY09XxQp"
+folder = "/output/MYE-251122_1929/eggd_multiqc_to_workbooks-1.0.1"
+file_search_term = ".*xlsx"
+
+# find the necessary files
+def find_files_in_project(project: dict, name: str,
```

Ruff (0.14.7)

Unused function argument: `project`

(ARG001)